<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegularExpressionValidator</class>
 <widget class="QWidget" name="RegularExpressionValidator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RegularExpressionValidator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_RegularExpression">
     <property name="placeholderText">
      <string>请在此输入正则表达式</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButton_Validate">
     <property name="text">
      <string>验证</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="comboBox_MatchRules"/>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTextEdit" name="textEdit_MatchContent"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
